lib/sheets.ts

```diff
@@ -0,0 +1,175 @@
+import * as fs from "fs";
+import { env } from "./core-exec";
+
+import { google } from 'googleapis';
+
+import { log } from './console';
+
+import { _await } from './core-exec';
+
+
+let token: any;
+let credentials: any;
+
+let oAuth2Client: any;
+let sheets: any;
+
+// parse token.js and credentials.js
+export function initialize() {
+    const tokenPath = env.GOOGLE_TOKEN_PATH;
+    const credPath = env.GOOGLE_CREDENTIAL_PATH;
+
+    const tokenFile = fs.readFileSync(tokenPath, 'utf8');
+    const credFile = fs.readFileSync(credPath, 'utf8');
+
+    token = JSON.parse(tokenFile);
+    credentials = JSON.parse(credFile);
+
+    const { client_secret, client_id, redirect_uris } = credentials.installed;
+    oAuth2Client = new google.auth.OAuth2(client_id, client_secret, redirect_uris[0]);
+    oAuth2Client.setCredentials(token);
+
+    sheets = google.sheets({ version: 'v4', oAuth2Client });
+
+    return;
+}
+
+/*
+
+function getNewToken(oAuth2Client) {
```

Comment:
We can remove this stub. It won't be necessary to test or in the final version of the module when we can instrument the OAuth2 flow. We'll handle this in the Orchestrator.